app/uk/gov/hmrc/tradergoodsprofiles/controllers/CreateRecordController.scala

```diff
@@ -45,19 +42,13 @@ class CreateRecordController @Inject() (
   def createRecord(eori: String): Action[JsValue] =
     (authAction(eori) andThen validateHeaderAction).async(parse.json) { implicit request =>
       (for {
-        createRecordRequest <- validateBody(request)
-        response            <- sendCreate(eori, createRecordRequest)
+        response <- sendCreate(eori, request)
```

Comment:
Since this is the only action performed, can we just move all the contents in the `sendCreate` function to this function? Same applies for the rest of the controllers

Reply:
Something like this:
```
def createRecord(eori: String): Action[JsValue] =
  (authAction(eori) andThen validateHeaderAction).async(parse.json) { implicit request =>
    routerService.createRecord(eori, request).map {
      case Right(serviceResponse) => Created(toJson(serviceResponse))
      case Left(error) => Status(error.status)(toJson(error.errorResponse))
    }
  }
```

Reply:
I have made the requested changes